pages/Teams/team-profile-page.cfm

```diff
@@ -54,6 +54,41 @@
   GROUP BY lastName, firstName
 </cfquery>
 
+<cfquery name="getTeamDataM" datasource="roundleague"> 
```

Comment:
I would say change this cfquery name to something that is more useful / meaningful